### DISCUSSION TITLE
Velero schedules for etcd backup

### DISCUSSION THREAD

**Discussion by cgatjens** [vmware-tanzu/velero#8361] [Community support Q&A]
Hi Team,

We are currently using Velero and MinIO buckets to backup our clusters, however we want to explore the option to backup etcd snapshots and PVs. Does Velero take backups of etcd that we can use to restore a cluster to its previous state if something breaks?

If so, what would be the options we need to include in Velero schedules section to backup etcd?

**Comment by Lyndon-Li** [comment]
Velero doesn't support to back up etcd at present.
Would you share more details of your requirements? Why would you want the etcd backup option over the namespace backups?

**Reply by cgatjens** [reply]
Thanks for your response. We are just working on different strategies to store our etcd snapshots on external solutions, and we currently use Velero to backup namespaces. The requirements are basically to have an automated way to create etcd backups and store them externally, but as per your response, I understand Velero is not the right solution as etcd backups are not supported.
